src/nft/pages/sell/sell.tsx

```diff
@@ -1,5 +1,85 @@
+import { useWeb3React } from '@web3-react/core'
+import { Box } from 'nft/components/Box'
+import { Center, Column, Row } from 'nft/components/Flex'
+import { ChevronLeftIcon, XMarkIcon } from 'nft/components/icons'
+import { ListPage } from 'nft/components/sell/list/ListPage'
+import { SelectPage } from 'nft/components/sell/select/SelectPage'
+import { buttonMedium, header2, headlineSmall } from 'nft/css/common.css'
+import { themeVars } from 'nft/css/sprinkles.css'
+import { useBag, useNFTList, useSellAsset, useSellPageState, useWalletCollections } from 'nft/hooks'
+import { ListingStatus, SellPageStateType } from 'nft/types'
+import { useEffect } from 'react'
+import { useNavigate } from 'react-router-dom'
+import { useToggleWalletModal } from 'state/application/hooks'
+
+import * as styles from './sell.css'
+
 const Sell = () => {
-  return <div>Sell NFTs</div>
+  const sellPageState = useSellPageState((state) => state.state)
+  const setSellPageState = useSellPageState((state) => state.setSellPageState)
+  const removeAllMarketplaceWarnings = useSellAsset((state) => state.removeAllMarketplaceWarnings)
+  const reset = useSellAsset((state) => state.reset)
+  const clearCollectionFilters = useWalletCollections((state) => state.clearCollectionFilters)
+  const setListingStatus = useNFTList((state) => state.setListingStatus)
+  const navigate = useNavigate()
+
+  useEffect(() => {
+    removeAllMarketplaceWarnings()
+    setListingStatus(ListingStatus.DEFINED)
+  }, [removeAllMarketplaceWarnings, sellPageState, setListingStatus])
+
+  const { account } = useWeb3React()
+  const toggleWalletModal = useToggleWalletModal()
+
+  useEffect(() => {
+    reset()
+    setSellPageState(SellPageStateType.SELECTING)
+    clearCollectionFilters()
+  }, [account, reset, setSellPageState, clearCollectionFilters])
+  const cartExpanded = useBag((state) => state.bagExpanded)
+
+  const exitSellFlow = () => {
+    navigate(-1)
+  }
+
+  return (
+    <Box className={styles.mobileSellWrapper}>
+      {/* <Head> TODO: figure out metadata tagging
+          <title>Genie | Sell</title>
+        </Head> */}
+      <Row className={styles.mobileSellHeader}>
+        {sellPageState === SellPageStateType.LISTING && (
+          <Box marginRight="4" onClick={() => setSellPageState(SellPageStateType.SELECTING)}>
+            <ChevronLeftIcon height={28} width={28} />
+          </Box>
+        )}
+        <Box className={headlineSmall} paddingBottom="4" style={{ lineHeight: '28px' }}>
+          {sellPageState === SellPageStateType.SELECTING ? 'Select NFTs' : 'Create Listing'}
+        </Box>
+        <Box cursor="pointer" marginLeft="auto" marginRight="0" onClick={exitSellFlow}>
+          <XMarkIcon height={28} width={28} fill={themeVars.colors.blackBlue} />
+        </Box>
+      </Row>
+      {account != null ? (
+        <Box style={{ width: `calc(100% - ${cartExpanded ? 324 : 0}px)` }}>
```

Comment:
nit: if the 324 is found in a diff part of the code, would recommend making a shared constant for it, or structuring the box differently